README.rst

```diff
@@ -1,5 +1,6 @@
-.. note:: Installs using pip or setup.py are not presently implemented.
-   Suggest manually add a tool to ``site_scons/site_tools`` to use.
+.. note:: Installs using pip searching pypi.org are not presently
+   implemented.  For alternative options, see the installation
```

Comment:
Not sure why you removed the note that site_scons/site_tools was a reasonable install location. I'll re-add post merge.

Reply:
I was thinking that pointing to the Installation section was sufficient since those updates should layout all the options.

Reply:
I figure always easier to provide info at point of use.
A great deal of users never read the docs, or enough of the docs for what they're trying to do.